packages/vscode-extension/package-lock.json

```diff
@@ -4,55 +4,6 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
```

Comment:
Why is package-lock.json changed?

Reply:
After updating the base branch `dev` and run `npm run setup`, this change is auto-done. Are we supposed to add the changes?

Reply:
Removed the changes.